src/hotspot/share/opto/loopnode.hpp

```diff
@@ -1218,19 +1255,29 @@ class PhaseIdealLoop : public PhaseTransform {
     Node* n = _idom[didx];
     assert(n != nullptr,"Bad immediate dominator info.");
     while (n->in(0) == nullptr) { // Skip dead CFG nodes
+      // We encountered a dead CFG node.
+      // If everything went right, this dead CFG node should have had a idom/ctrl
```

Comment:
In the context of idom, you might be able to remove "ctrl":
```suggestion
      // If everything went right, this dead CFG node should have had an idom
```